CMakeLists.txt

```diff
@@ -454,16 +466,21 @@ endif()
 #  target_link_libraries(executableName ${libpointmatcher_LIBRARIES})
 #  ...
 
-# 1- local build #
+# Install cmake config module
+install(EXPORT ${PROJECT_NAME}-config DESTINATION share/${PROJECT_NAME}/cmake)
 
-# Register the local build in case one doesn't use "make install"
-export(PACKAGE libpointmatcher)
+# make project importable from build_dir:
+export(
+	TARGETS ${LIB_NAME}
+	FILE ${CMAKE_BINARY_DIR}/${PROJECT_NAME}-config.cmake
+)
+add_library(${PROJECT_NAME}::${PROJECT_NAME} ALIAS ${LIB_NAME})
```

Comment:
Would it be possible to instead reuse PROJECT_NAME?

Reply:
I guess you refer to the LIB_NAME? Notice my comment above: one has the "lib" prefix, the other doesn't.